webclient/webclient/src/main/java/io/helidon/webclient/Proxy.java

```diff
@@ -59,6 +60,10 @@
     private static final Pattern IP_V6_HOST = Pattern.compile("^(?:[0-9a-fA-F]{1,4}:){7}[0-9a-fA-F]{1,4}$");
     private static final Pattern IP_V6_HEX_HOST = Pattern
             .compile("^((?:[0-9A-Fa-f]{1,4}(?::[0-9A-Fa-f]{1,4})*)?)::((?:[0-9A-Fa-f]{1,4}(?::[0-9A-Fa-f]{1,4})*)?)$");
+
+    private static final LruCache<String, Boolean> IVP6_HOST_MATCH_RESULTS = LruCache.create();
```

Comment:
Default capacity for LruCache is 10,000 which seems much larger than is needed?

Reply:
Good point. I will change that.

Reply:
fixed